### ISSUE TITLE
Document how SSA works

### ISSUE THREAD

**Issue by sthaha** [sustainable-computing-io/kepler-operator#196]

**Comment by Obasoro** [comment]
@sthaha is this still open?

**Comment by sthaha** [comment]
@Obasoro absolutely, please feel free to grab it if you are inclined :). Thanks a lot for helping out.

**Comment by Obasoro** [comment]
Thank you @sthaha

**Comment by Obasoro** [comment]
@sthaha gone through the SSA docs, I want to confirm the right docs for [SSA](https://github.com/rhobs/observability-operator/blob/main/docs/user-guides/server-side-apply.md).

**Comment by sthaha** [comment]
@Obasoro , that is only to serve as an example for SSA and what contents to include in the SSA doc. In Keplers case, the resources created in response to user creating `kepler` is going to be different however SSA can be used to modify fields not managed by the `kepler` controller.
